src/scipp/html/formatting_html.py

```diff
@@ -266,9 +268,11 @@ def _make_dim_labels(dim, bin_edges=None):
 
 def _make_dim_str(var, bin_edges, add_dim_size=False):
     dims_text = ', '.join(
-        '{}{}{}'.format(str(dim), _make_dim_labels(dim, bin_edges),
+        '{}{}{}'.format(str(dim),
+                        _make_dim_labels(dim, bin_edges),
                         f': {size}' if add_dim_size and size is not None else '')
-        for dim, size in zip(var.dims, var.shape))
+        for dim,
+        size in zip(var.dims, var.shape))
```

Comment:
This is still happening!

Reply:
Yeah, i think it's because of `split_all_top_level_comma_separated_values`...